src/manager.js

```diff
@@ -19,14 +19,17 @@ $('#future-bookings').hide();
 let bookingData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/bookings/bookings')
   .then(data => data.json())
   .then(data => data.bookings)
+  .catch(data => console.log('booking data error', data))
```

Comment:
I haven't implemented this yet but have you considered displaying an error message on the DOM rather than just console.logging?